connections/ethereum/connection_test.go

```diff
@@ -98,11 +103,55 @@ func TestConnection_SafeEstimateGasMax(t *testing.T) {
 	}
 }
 
+func TestConnection_SafeEstimateGasMin(t *testing.T) {
+	minPrice := big.NewInt(1)
+	// When gasMultipler is zero, the gasPrice is zero if the effect of the minPrice is removed.
+	gasMultipler := big.NewFloat(0)
+	conn := NewConnection(TestEndpoint, false, AliceKp, log15.Root(), GasLimit, MaxGasPrice, minPrice, gasMultipler, "", "")
+	err := conn.Connect()
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer conn.Close()
+
+	price, err := conn.SafeEstimateGas(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if price.Cmp(minPrice) != 0 {
+		t.Fatalf("Gas price should equal min. Suggested: %s Min: %s", price.String(), minPrice.String())
+	}
+}
+
+func TestConnection_SafeEstimateGasSameMin(t *testing.T) {
+	minPrice, maxGasPrice := big.NewInt(0), big.NewInt(0)
+	// When GasMultipler is set to 1, the gas price is set to 2000000000, so the minPrice is set to the same price
+	// and maxPrice is made larger than the minPrice by adding one.
+	minPrice.Add(MaxGasPrice, big.NewInt(0))
```

Comment:
Can't you just set `minPrice` to `MaxGasPrice` right away when it is declared and delete this line

Reply:
thank you
I fixed it https://github.com/ChainSafe/ChainBridge/pull/689/commits/514ca0bbc012e66f66eb93f5dffed5e14f896ee3.